pyk/src/pyk/integration_tests/test_kprove.py

```diff
@@ -103,4 +103,4 @@ def _config(k: str, state: str) -> CTerm:
         init = _config('if (B:Bool) { $n = 2;  } else { $n = 3; }', '$n |-> 1')
         final = _config('.K', '$n |-> 1')
         result = self.kprove.prove_cterm('prove-cterm-multiple-results', init, final)
-        self.assertEqual(len(result), 2)
+        self.assertEqual(len(result), 1)  # Should be 2
```

Comment:
I expected `_config('.K', '$n |-> 2') #Or _config('.K', '$n |-> 3')` but only got `n |-> 2` (even when running kprove from the command line directly).